zowex/src/main.rs

```diff
@@ -181,14 +263,19 @@ fn run_daemon_command(mut args: String) -> std::io::Result<()> {
     Ok(talk(&_resp, &mut stream)?)
 }
 
+/**
+ * Attempt to make a TCP connection to the daemon.
+ * Iterate to enable a slow system to start the daemon.
+ */
 fn establish_connection(host: String, port: String) -> std::io::Result<TcpStream> {
-    /* Attempt to make a TCP connection to the daemon.
-     * Iterate to enable a slow system to start the daemon.
-     */
+    const THREE_MIN_OF_RETRIES: i32 = 60;
```

Comment:
Can we also add a variable for the 3 seconds for the `sleep` function ?
That may make it a little more clear as to how we get the 3 minutes 😋

Reply:
Good idea. I am surprised that the compiler did not complain about my hard-coded 3.